scripts/compare_benchmarks.py

```diff
@@ -11,15 +11,15 @@
 min_iterations = 10
 min_runtime_ns = 59 * 1000 * 1000 * 1000
```

Comment:
With using the runtimes, do we now automatically cover the shuffled issue as well? Should be the case, right?

Reply:
Well, the information continues to be pointless, but at least it should not be misleading anymore. It doesn't fix the issue described in #1972 however.

Reply:
Do you consider #1972 to be fixed now? If so, feel free to set the TODO marker in the PR description.